src/jira/client/index.ts

```diff
@@ -361,25 +363,31 @@ export const getTruncatedIssuekeys = (data: IssueKeyObject[] = []): IssueKeyObje
  * with issue keys in a Jira Repository object
  */
 const updateRepositoryIssueKeys = (repositoryObj, mutatingFunc) => {
-	if (repositoryObj.commits) {
-		repositoryObj.commits = updateIssueKeysFor(
-			repositoryObj.commits,
-			mutatingFunc
-		);
-	}
-	if (repositoryObj.branches) {
-		repositoryObj.branches = updateIssueKeysFor(
-			repositoryObj.branches,
-			mutatingFunc
-		);
-		repositoryObj.branches.forEach((branch) => {
-			if (branch.lastCommit) {
-				branch.lastCommit = updateIssueKeysFor(
-					[branch.lastCommit],
-					mutatingFunc
-				)[0];
-			}
-		});
+	const logger = getLogger("jiraClient.repository.update");
```

Comment:
Can we pass logger as a function parameter instead of creating a new one here? (Otherwise we won't be able to connect it by requestId/ jobId)